tests/test_types.py

```diff
@@ -1422,16 +1422,34 @@ class Model(BaseModel):
         Model(a=6)
 
 
-def test_number_multiple_of():
+@pytest.mark.parametrize('value,is_valid', ((10, True), (1337, False), (23, False), (100, True), (20, True)))
```

Comment:
cleaner I think to have two tests `test_number_multiple_of_int_valid` and `test_number_multiple_of_int_invalid`